src/slide-image-source.cc

```diff
@@ -0,0 +1,159 @@
+// -*- mode: c++; c-basic-offset: 4; indent-tabs-mode: nil; -*-
+// (c) 2021 Leonardo Romor <leonardo.romor@gmail.com>,
+//          Henner Zeller <h.zeller@acm.org>
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation version 2.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://gnu.org/licenses/gpl-2.0.txt>
+
+#include "slide-image-source.h"
+#include "framebuffer.h"
+#include "terminal-canvas.h"
+
+#include <cassert>
+#include <cstdio>
+#include <unistd.h>
+#include <openslide/openslide.h>
```

Comment:
If you have it in this group it should probably be sorted (so after cstdio and before unistd).

Alternatively, add a newline above so that it is its own 'include group'

Reply:
(solved)